Automattic/wp-calypso
# My plan: add Site Activity card

#### Changes proposed in this Pull Request

* Adds an Activity card to the admin page.
* Related discussion: p9rlnk-yi-p2

#### Visuals

![image](https://user-images.githubusercontent.com/390760/47355664-dd9abe80-d6b9-11e8-8ff0-539d29c65030.png)

#### Testing instructions

* Fire up this PR.
* Go to My Plans page: `/plans/my-plan/[site_url]`.
* Make sure you see the card, and that the button is working correctly.
